docs/08-MultiPageDashApp.md

```diff
@@ -285,47 +323,63 @@ page navigation links or by template authors.
 - `module`:
    The module path where this page's `layout` is defined. Often `__name__`.
 
+
 - `path`:
    URL Path, e.g. `/` or `/home-page`.
    If not supplied, will be inferred from `module`,
    e.g. `pages.weekly_analytics` to `/weekly-analytics`
 
+
+- path_template:
+       Add variables to a URL by marking sections with <variable_name>. The layout function
+       then receives the <variable_name> as a keyword argument.
+       e.g. path_template= "/asset/<asset_id>"
+            if pathname = "/assets/a100" then layout will receive {"asset_id":"a100"}
```

Comment:
```suggestion
            if pathname = "/assets/a100" then layout will receive **{"asset_id": "a100"}
```

This doc string makes it sound like the dictionary is passed directly into the first argument of layout, when really we pass in an unpacked set of keyword arguments. 

not sure if ** will be clear to users. maybe we say "will receive asset_id="a100""